src/ResourceManager/Sql/Commands.Sql.Test/ScenarioTests/DataMaskingTests.cs

```diff
@@ -25,12 +25,13 @@ public class DataMaskingTests : SqlTestsBase
     {
         protected override void SetupManagementClients(RestTestFramework.MockContext context)
         {
-            var sqlCSMClient = GetSqlClient();
+            var sqlClient = GetSqlClient(context);
```

Comment:
Understood. I appreciate the feedback on potential code cleanups, I just want to be clear that since this PR is already huge I don't want to change any more code than is necessary. :)